frontend/src/components/Message/Message.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <!-- Header -->
+    <div class="header py-lg-6">
+      <b-container>
+        <div class="header-body text-center mb-7">
+          <!-- eslint-disable-next-line @intlify/vue-i18n/no-dynamic-keys-->
+          <p class="h1 test-message-headline">{{ $t(headline) }}</p>
```

Comment:
as we discussed before, this method will have the problem that the compiler will not be able to determine the used text by itself. Consider to have the property be the actual text already, resolved by `$t` so that the localization will not be task of this component.

Reply:
Thanks for the hint.
I left the `$t` here unintentional and faulty.